### ISSUE TITLE
ITK DLL/so required is ITK 5.2, but the package depends on ITK 5.3

### ISSUE THREAD

**Issue by KrisThielemans** [conda-forge/petpvc-feedstock#4]
### Solution to issue cannot be found in the documentation.

- [X] I checked the documentation.

### Issue

Running any of the PETPVC executables currently doesn't work. On Windows, nothing happens at all. On Linux I get
```
$ pvc_make4d
pvc_make4d: error while loading shared libraries: libITKIOBruker-5.2.so.1: cannot open shared object file: No such file or directory
```
On Windows, checking the DLL dependencies, I see similarly
```
dumpbin /dependents .\pvc_make4d.exe
Microsoft (R) COFF/PE Dumper Version 14.29.30138.0
Copyright (C) Microsoft Corporation.  All rights reserved.


Dump of file .\pvc_make4d.exe

File Type: EXECUTABLE IMAGE

  Image has the following dependencies:

    ITKIOBruker-5.2.dll
    ITKIOHDF5-5.2.dll
    ITKIOJPEG2000-5.2.dll
    <snip>
```
However, `conda list` confirms the package depends on `libitk` 5.3.0. Indeed, the DLLs in the `miniconda3\envs\petpvc\Library\bin` are the 5.3 ones (and some on Linux).

I don't quite understand how this can happen, but maybe a re-render that went wrong?

### Installed packages

```shell
This is after installing libitk-devel as well, but that doesn't really make any difference
# packages in environment at C:\Users\krisf\miniconda3\envs\petpvc:
#
# Name                    Version                   Build  Channel
ca-certificates           2022.12.7            h5b45459_0    conda-forge
eigen                     3.4.0                h2d74725_0    conda-forge
fftw                      3.3.10          nompi_h38027f0_106    conda-forge
hdf5                      1.12.2          nompi_h57737ce_101    conda-forge
jpeg                      9e                   hcfcfb64_3    conda-forge
krb5                      1.20.1               heb0366b_0    conda-forge
lerc                      4.0.0                h63175ca_0    conda-forge
libaec                    1.0.6                h63175ca_1    conda-forge
libcurl                   7.88.1               h68f0423_0    conda-forge
libdeflate                1.17                 hcfcfb64_0    conda-forge
libhwloc                  2.9.0                h51c2c0f_0    conda-forge
libiconv                  1.17                 h8ffe710_0    conda-forge
libitk                    5.3.0                h2d081bb_2    conda-forge
libitk-devel              5.3.0                h002f227_2    conda-forge
libssh2                   1.10.0               h9a1e1f7_3    conda-forge
libtiff                   4.5.0                hf8721a0_2    conda-forge
libxml2                   2.10.3               hc3477c8_0    conda-forge
libzlib                   1.2.13               hcfcfb64_4    conda-forge
openssl                   3.0.8                hcfcfb64_0    conda-forge
petpvc                    1.2.10               h2d74725_0    conda-forge
pthreads-win32            2.9.1                hfa6e2cd_3    conda-forge
tbb                       2021.8.0             h91493d7_0    conda-forge
tbb-devel                 2021.8.0             h91493d7_0    conda-forge
ucrt                      10.0.22621.0         h57928b3_0    conda-forge
vc                        14.3                hb6edc58_10    conda-forge
vs2015_runtime            14.34.31931         h4c5c07a_10    conda-forge
xz                        5.2.6                h8d14728_0    conda-forge
zstd                      1.5.2                h12be248_6    conda-forge
```

### Environment info

```shell
active environment : base
    active env location : C:\Users\krisf\miniconda3
            shell level : 1
       user config file : C:\Users\krisf\.condarc
 populated config files : C:\Users\krisf\.condarc
          conda version : 23.1.0
    conda-build version : 3.23.3
         python version : 3.8.15.final.0
       virtual packages : __archspec=1=x86_64
                          __win=0=0
       base environment : C:\Users\krisf\miniconda3  (writable)
      conda av data dir : C:\Users\krisf\miniconda3\etc\conda
  conda av metadata url : None
           channel URLs : https://conda.anaconda.org/conda-forge/win-64
                          https://conda.anaconda.org/conda-forge/noarch
                          https://repo.anaconda.com/pkgs/main/win-64
                          https://repo.anaconda.com/pkgs/main/noarch
                          https://repo.anaconda.com/pkgs/r/win-64
                          https://repo.anaconda.com/pkgs/r/noarch
                          https://repo.anaconda.com/pkgs/msys2/win-64
                          https://repo.anaconda.com/pkgs/msys2/noarch
          package cache : C:\Users\krisf\miniconda3\pkgs
                          C:\Users\krisf\.conda\pkgs
                          C:\Users\krisf\AppData\Local\conda\conda\pkgs
       envs directories : C:\Users\krisf\miniconda3\envs
                          C:\Users\krisf\.conda\envs
                          C:\Users\krisf\AppData\Local\conda\conda\envs
               platform : win-64
             user-agent : conda/23.1.0 requests/2.28.2 CPython/3.8.15 Windows/10 Windows/10.0.19044
          administrator : False
             netrc file : None
           offline mode : False
```

**Comment by KrisThielemans** [comment]
I hope to release a bug-fix version of PETPVC tomorrow, so maybe this will be fixed automatically when updating this feedstock accordingly

**Comment by KrisThielemans** [comment]
@ghisvail maybe you want to make me co-maintainer?

**Comment by KrisThielemans** [comment]
https://github.com/UCL/PETPVC/releases/tag/v1.2.11 is now released

**Comment by KrisThielemans** [comment]
@ghisvail This seems to be fixed in #5.  

It is possible that this was caused by a bug in the itk feedstock, which was fixed in https://github.com/conda-forge/libitk-feedstock/commit/04e8572bf653e0544b73c2295db8200e08c12b3a

**Comment by KrisThielemans** [comment]
👍
